agent/engine/engine_windows_integ_test.go

```diff
@@ -68,18 +72,14 @@ func createTestHostVolumeMountTask(tmpPath string) *apitask.Task {
 	return testTask
 }
 
-func createTestEmptyHostVolumeMountTask() *apitask.Task {
-	testTask := createTestTask("testEmptyHostVolumeMount")
-	testTask.Volumes = []apitask.TaskVolume{{Name: "test-tmp", Volume: &apitask.EmptyHostVolume{}}}
+func createTestLocalVolumeMountTask(tmpPath string) *apitask.Task {
+	testTask := createTestTask("testLocalHostVolumeMount")
 	testTask.Containers[0].Image = testVolumeImage
-	testTask.Containers[0].MountPoints = []apicontainer.MountPoint{{ContainerPath: "C:/empty", SourceVolume: "test-tmp"}}
-	testTask.Containers[0].Command = []string{`While($true){ if (Test-Path C:\empty\file) { exit 42 } }`}
-	testTask.Containers = append(testTask.Containers, createTestContainer())
-	testTask.Containers[1].Name = "test2"
-	testTask.Containers[1].Image = testVolumeImage
-	testTask.Containers[1].MountPoints = []apicontainer.MountPoint{{ContainerPath: "C:/alsoempty/", SourceVolume: "test-tmp"}}
-	testTask.Containers[1].Command = []string{`New-Item -Path C:\alsoempty\file`}
-	testTask.Containers[1].Essential = false
+	testTask.Containers[0].Command = []string{`echo -n empty-data-volume | Out-File -FilePath C:\host\tmp\hello-from-container -Encoding ascii`}
+	testTask.Containers[0].MountPoints = []apicontainer.MountPoint{{ContainerPath: "C:\\host\\tmp", SourceVolume: "test-tmp"}}
```

Comment:
Also, "C:\host\tmp\" is fine for testing use. But a more traditional path for powershell/windows would be "$ENV:TEMP" for the current user's temp dir. I would consider using this so that running the tests dont create unnecessary and unconventional directories on the root volume